### ISSUE TITLE
Console gets mixedup after specific output

### ISSUE THREAD

**Issue by tiagocoutinho** [vxgmichel/aioconsole#26]
Executing any python code that results in printing to the screen a line that ends with  `'. \n'` or `'> \n'` messes up the console.
Example: printing any string on the screen that ends with `'. \n'` or `'> \n'` messes up the console:

```
$ apython
>>> print('> ')
> # cursor stays here as if waiting from a user prompt. 

# From now on the console is in a weird state
```

Expected:
```
$ apython
>>> print('> ')
> 
>>> # I get the prompt back
```

**Comment by tiagocoutinho** [comment]
Another example (maybe is a different issue) is the printing with `print(..., end='', flush=True)`

Example:
```
$ apython
>>> for i in range(10):
...   print('#', end='\t', flush=True)
...   # console gets blocked here
```

**Comment by vxgmichel** [comment]
Thanks for the report @tiagocoutinho 

Those two issues are related to the hackish prompt detection for the readline support, but it should get better once PR #25 is merged.

For the moment, the simplest way to work around those issues is to disable readline: `apython --no-readline`.

**Comment by vxgmichel** [comment]
Fixed! It will become visible in version 0.1.9.
